teacher.rb

```diff
@@ -0,0 +1,35 @@
+require_relative 'dbc'
+
+class Teacher < DBCPerson
+	attr_reader :salary, :target_raise
+
+	def initialize(options={})
+		super
+		@target_raise = 0
+	end
+
+	def set_performance_rating(rating)
+   	response = ""
+   	if rating > self.class::RATING
+   		response = "Yay, I'm a great employee!"
+     	 	receive_raise(@target_raise)
+   	else
+      	response += "Oh, well -- thanks to this actionable, specific, and kind "
+      	response += "feedback, I'll do better next time."
+    	end
+    	response
+  	end	
+
+  	def teach_stuff
+		response = ""
+	 	response += self.class::TEACH_RESPONSE
+	   response += self.class::BOMB_RESPONSE
+	   response += self.class::SASSY
+	   response
+   end

```

Comment:
Indentation may seem trivial, as the program works whether you're following standard conventions or not.  But when you're working on a large project with lots of other people, you want to know that everyone is following the same style guide.  Even just looking back on your own code, you'll prefer well formatted code over poorly formatted code!  And any workplace with code reviews in place will require that you follow their style guide, so get used to it now.

Make sure to set sublime (or whatever text editor you're using) to have 2-space tabs, since that's what we use in the files we give you, and we like consistency (also, sometimes if you have tabs in some places and spaces in others, it can look right to you, but then look wrong on github).

http://stackoverflow.com/questions/9474090/how-do-i-force-sublime-text-2-to-indent-two-spaces-per-tab